storage/rocksdb/rdb_datadic.cc

```diff
@@ -3137,6 +3145,19 @@ bool Rdb_tbl_def::put_dict(Rdb_dict_manager *const dict,
   return false;
 }
 
+uint32 Rdb_key_def::generate_index_flags_bitmap(const Rdb_key_def &kd) {
+  return (kd.has_ttl() ? Rdb_key_def::TTL_FLAG : 0);
+}
+
+uint32 Rdb_key_def::calculate_index_flag_offset(uint32 index_flags,
+                                                enum INDEX_FLAG flag) {
+  /*
+    TODO(alexyang): Always return 0 for now as TTL is the only index flag,
```

Comment:
I left it out for now since TTL is the only index flag, it will always return 0.  But I can add some skeleton logic in there :)